deps.macos/80-asio.zsh

```diff
@@ -0,0 +1,25 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='asio'
+local version='1.12.1'
+local url='https://github.com/chriskohlhoff/asio.git'
+local hash='b73dc1d2c0ecb9452a87c26544d7f71e24342df6'
+
+## Build Steps
+setup() {
+  log_info "Setup (%F{3}${target}%f)"
+  setup_dep ${url} ${hash}
+}
+
+install() {
+  autoload -Uz progress
+
+  log_info "Install (%F{3}${target}%f)"
+  cd "${dir}/asio"
+
+  mkdir -p ${target_config[output_dir]}/include
+
+  log_output "include => ${target_config[output_dir]}/include"
```

Comment:
I would move this below the copy command and have it say `log_status "Copied headers to ${target_config[output_dir]}"`.

Usually I went with `log_debug` before doing things and `log_status` after it was successful.

Reply:
Made the change.